packages/jest-config/src/normalize.js

```diff
@@ -185,9 +185,12 @@ function normalize(config, argv) {
   if (config.scriptPreprocessor) {
     config.scriptPreprocessor =
       _replaceRootDirTags(config.rootDir, config.scriptPreprocessor);
-    if (config.scriptPreprocessor.includes(
-      constants.NODE_MODULES + 'babel-jest',
-    )) {
+    if (
+      config.scriptPreprocessor.includes(
+        constants.NODE_MODULES + 'babel-jest',
+      )
+      || config.scriptPreprocessor.includes('packages/babel-jest')
```

Comment:
agree and i actually already implemented it, but then rolled back because PR scope started growing to fast.

the only thing is in this specific file the preprocessor is not yet required and we're only dealing with the `Path`.
I'll come back to this in one of the next PRs